labgrid/util/helper.py

```diff
@@ -26,21 +27,34 @@ def get_user():
     import pwd
     return pwd.getpwuid(os.getuid())[0]
 
+def set_nonblocking(fd):
+    flags = fcntl.fcntl(fd, fcntl.F_GETFL)
+    fcntl.fcntl(fd, fcntl.F_SETFL, flags | os.O_NONBLOCK)
+
 @attr.s
 class ProcessWrapper:
     callbacks = attr.ib(default=attr.Factory(list))
     loglevel = logging.INFO
 
     @step(args=['command'], result=True, tag='process')
-    def check_output(self, command, *, print_on_silent_log=False):
+    def check_output(self, command, *, print_on_silent_log=False, input=None):
```

Comment:
```suggestion
    def check_output(self, command, *, print_on_silent_log=False, input=None):  # pylint: disable=redefined-builtin
```
Or is there variable name you could use instead? If not, just disable the warning.

Reply:
`input` is the name of the argument in python `subprocess.check_output()` so I think we want to keep it the same?

Reply:
Yeah, disabling the warning is the correct move here, we want to keep it as is.

Reply:
Agreed.